test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -90,29 +98,30 @@ function pushImgToNodes() {
 
     echo "Copying $IMG_NAME image to every node..."
     # Copy image to control-plane node
-    scp -F ssh-config $SAVED_IMG k8s-node-control-plane:/tmp/antrea-ubuntu.tar &
+    scp -F ssh-config $SAVED_IMG k8s-node-control-plane:/tmp/image.tar &
     pids[0]=$!
     # Loop over all worker nodes and copy image to each one
     for ((i=1; i<=$NUM_WORKERS; i++)); do
         name="k8s-node-worker-$i"
-        scp -F ssh-config $SAVED_IMG $name:/tmp/antrea-ubuntu.tar &
+        scp -F ssh-config $SAVED_IMG $name:/tmp/image.tar &
         pids[$i]=$!
     done
     # Wait for all child processes to complete
     waitForNodes "${pids[@]}"
     echo "Done!"
 
     echo "Loading $IMG_NAME image in every node..."
-    ssh -F ssh-config k8s-node-control-plane docker load -i $SAVED_IMG &
+    ssh -F ssh-config k8s-node-control-plane docker load -i /tmp/image.tar &
     pids[0]=$!
     # Loop over all worker nodes and copy image to each one
     for ((i=1; i<=$NUM_WORKERS; i++)); do
         name="k8s-node-worker-$i"
-        ssh -F ssh-config $name docker load -i $SAVED_IMG &
+        ssh -F ssh-config $name docker load -i /tmp/image.tar &
         pids[$i]=$!
     done
     # Wait for all child processes to complete
     waitForNodes "${pids[@]}"
+    rm -f $SAVED_IMG
```

Comment:
can't we delete /tmp/image.tar as well after loading is done on the nodes?

Reply:
We can, it will help save some disk space on the nodes. Done.